samples/suit/recovery/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+if (NOT SYSBUILD)
+	message(WARNING
+		" This is a multi-image application that should be built using sysbuild.\n"
+		" Add --sysbuild argument to west build command to prepare all the images.")
+endif()
```

Comment:
It this warning needed? I think only the multicore/hello_world has something like this.

Reply:
Removed